core/src/main/java/de/evoila/cf/broker/controller/core/ServiceInstanceBindingController.java

```diff
@@ -47,31 +56,84 @@ public ServiceInstanceBindingController(BindingServiceImpl bindingService) {
 		if (request.getAppGuid() != null && request.getAppGuid().isEmpty())
             return new ResponseEntity("{}", HttpStatus.BAD_REQUEST);
 
-		ServiceInstanceBindingResponse response = bindingService.createServiceInstanceBinding(bindingId, instanceId, request);
+		if (acceptsIncomplete == null){
+			acceptsIncomplete = false;
+		}
+		if (acceptsIncomplete && apiHeader.equals("2.13")){
```

Comment:
We should extract the API Versions 2.13 and 2.14 to a Constant Class ApiVersion.java